packages/shader-lab/src/preprocessor/PpParser.ts

```
@@ -642,20 +644,21 @@
-      const startPosition = ShaderLab.createPosition(start);
-      const endPosition = ShaderLab.createPosition(end);
-      const range = ShaderLab.createRange(startPosition, endPosition);
-      this.expandSegments.push({ rangeInBlock: range, replace: "" });
-    }
+    const start = scanner.current - token.lexeme.length;
+    scanner.scanPairedBlock("{", "}");
+    const end = scanner.current;
+    const startPosition = ShaderLab.createPosition(start);
+    const endPosition = ShaderLab.createPosition(end);
+    const range = ShaderLab.createRange(startPosition, endPosition);
+    this.expandSegments.push({ rangeInBlock: range, replace: "" });
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)